config/locales/zh-CN.yml

```diff
@@ -92,10 +92,12 @@ zh-CN:
         expires_at: "过期日期"
         name: "名称"
         path: "路径"
+        promotion_category: "促销分类"
```

Comment:
`promotion_category ` do not exists in core en.yml file, but it is used in promotions form of spree_backend. See: https://github.com/spree/spree/blob/master/backend/app/views/spree/admin/promotions/_form.html.erb#L35 . This label displays right with default translation in English, but wrong in Chinese.